gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinTaskRunner.java

```diff
@@ -599,6 +601,7 @@ public void run() {
   private Optional<ContainerMetrics> buildContainerMetrics() {
     Properties properties = ConfigUtils.configToProperties(this.clusterConfig);
     if (GobblinMetrics.isEnabled(properties)) {
+      logger.info("Container metrics are enabled");
```

Comment:
Nice log addition! Do you know where these container metrics are collected?

Reply:
The `METRICS_KAFKA_TOPIC` and other Kafka related metrics parameters in `ConfigurationKeys` are what we use to define where the `ScheduledReporter` outputs metrics.